app.py

```diff
@@ -15,45 +15,46 @@ def index():
     # If form is submitted
     if request.method == 'POST':
         NotamRequest = Models.NotamRequest(request.form)
-
-        # get lat/long of airports
-        NotamRequest.startLat, NotamRequest.startLong = alc.get_lat_and_lon(NotamRequest.startAirport)
-        NotamRequest.destLat, NotamRequest.destLong = alc.get_lat_and_lon(NotamRequest.destAirport)
+        airports = [NotamRequest.startAirport, NotamRequest.destAirport, NotamRequest.additionalAirport1, NotamRequest.additionalAirport2, NotamRequest.additionalAirport3 ]
+        apiOutputs = []
+        i= 0
         
-        NotamRequest.radius = int(NotamRequest.radius)
-        NotamRequest.pathWidth = int(NotamRequest.pathWidth)
-
-        # get the list of coordinates that need to be called to cover area
-        coordList = MinimalCirclesPath.getPath(NotamRequest.startLat, 
-                                               NotamRequest.startLong,
-                                               NotamRequest.destLat,
-                                               NotamRequest.destLong, 
-                                               NotamRequest.radius, # circle radius
-                                               NotamRequest.pathWidth) # path width
+        while i != 4 and airports[i+1] is not None:
+            
+            # get lat/long of airports
+            NotamRequest.startLat, NotamRequest.startLong = alc.get_lat_and_lon(airports[i])
+            NotamRequest.destLat, NotamRequest.destLong = alc.get_lat_and_lon(airports[i+1])
+        
+            NotamRequest.radius = int(NotamRequest.radius)
+            NotamRequest.pathWidth = int(NotamRequest.pathWidth)
 
-        # start timer
-        startTime = time.time() 
+            # get the list of coordinates that need to be called to cover area
+            coordList = MinimalCirclesPath.getPath(NotamRequest.startLat, 
+                                                   NotamRequest.startLong,
+                                                   NotamRequest.destLat,
+                                                   NotamRequest.destLong, 
+                                                   NotamRequest.radius, # circle radius
+                                                   NotamRequest.pathWidth) # path width
 
-        # call the API for each point
-        print("LOADING...")
+            # start timer
+            startTime = time.time() 

```

Comment:
for testing purposes this should be on the outside of the while loop to time the entire process. An alternative could be moving the end to be inside the loop to time each loop individually. Having the start inside and the end outside the loop causes it to be overwritten multiple times before ending.

Reply:
this issue still stands even with the latest changes